cmd/minishift/cmd/artifact_preflight.go

```diff
@@ -29,7 +29,14 @@ import (
 
 // preflightChecksForArtifacts is executed once artifacts are cached.
 func preflightChecksForArtifacts() {
-	artifactsCheckSucceedsOrFails(checkOcFlag, "Checking if provided oc flags are supported", "Provided oc flag not supported")
+
+	// checking if the flags provided during start are supported by the oc binary
+	ocFlagCheckSucceedsOrFails(
```

Comment:
@LalatenduMohanty how about using the following function instead of duplicating? Then you can also get rid of that `ocFlagCheckSucceedsOrFails` function also and there will be no duplication.

```golang
// checking if the flags provided during start are supported by the oc binary
	preflightCheckSucceedsOrFails(
		configCmd.SkipCheckOcFlag.Name,
		checkOcFlag,
		"Checking if provided oc flags are supported",
		configCmd.WarnCheckOcFlag.Name,
		"Provided oc flag not supported")
```

Reply:
That's two with the same comment

Reply:
Right I should have reused the function. Will fix it.

Reply:
> That's two with the same comment

@gbraad Sorry did not get you. What do you want to say?